_projects/community-share.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Community Share
+date: 2015-06-01
+link: https://github.com/communityshare/communityshare
+intro: Come help with
+image: /assets/images/communityshare.png
+lead: Josh Schachter & Ben Reynwar
+contact: http://www.communityshare.us/contact/
```

Comment:
@seanastephens I'm commandeering this if you don't mind. Since we will hopefully redo this site, I'm going to let the impurity slip in - the link tag inside the markdown.